packages/client/tests/e2e/mongodb-notablescan/package-lock.json

```diff
@@ -0,0 +1,791 @@
+{
```

Comment:
This is the lockfile from `npm`, we should remove it.
I think you might've run `npm` instead of `pnpm` at some point.